src/pocketmine/Server.php

```diff
@@ -466,7 +466,7 @@ public function isHardcore() : bool{
 	 * @return int
 	 */
 	public function getDefaultGamemode() : int{
-		return $this->getConfigInt("gamemode", 0) & 0b11;
+		return GameMode::fromString($this->getConfigString("gamemode", "s"));
```

Comment:
I would be happier if `(string) GameMode::SURVIVAL` was used here. It's too easy to miss this if refactors happen.